source/library/Autoexporter.hs

```diff
@@ -2,12 +2,16 @@ module Autoexporter where
 
 import qualified Data.List as List
 import qualified Data.Maybe as Maybe
+import qualified Data.Traversable as Traversable
 import qualified Distribution.ModuleName as ModuleName
 import qualified Distribution.Text as Text
 import qualified System.Directory as Directory
 import qualified System.Environment as Environment
 import qualified System.FilePath as FilePath
 
+data ExportScope = ExportScope'Shallow
+                 | ExportScope'Deep
```

Comment:
I haven't seen this style of constructor names before. Can you remove the apostrophes (`ExportScopeShallow`)?